### DISCUSSION TITLE
articles/understanding-of-existentialism

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#117] [Comments]
https://hutusi.com/articles/understanding-of-existentialism 

 前段时间看了两本存在主义的入门书，一本是《存在主义咖啡馆》，一本是《非理性的人》。《存在主义咖啡馆》更像是一本传记类的书籍，主要笔墨都落在了存在主义的两位代表性人物身上：海德格尔和萨特，其他则是在写影响或受他们影响的人物：克尔凯郭尔、胡塞尔、雅斯贝尔斯、梅洛-庞蒂、阿尔贝·加缪、汉娜·阿伦特，以及西蒙娜·德·波伏娃。在读本书过程中，一度会认为它不是在写存在主义，而是在写萨特和波伏娃的轶事。但作者的观点是：思想很有趣，但人更有趣。作者年轻的时候学习哲学，沉浸于概念，不会去考虑它们跟创造概念的人之间的关系。而过了三十年等她写作此书时，她改变了以往的看法。在她看来，创造这些哲学概念的人他们所经历的各种事情才是最有趣的，当我们从书中去体验它们，会感受存在主义所带来的愤怒和活力，那些“那些美轮美奂、如磷光一般闪耀的繁盛”。

**Comment by c4710n** [comment]
感谢分享 ;)

**Comment by hutusi** [comment]
@c4710n 😀

**Comment by Rayme** [comment]
读完了很有收获，马上开始阅读这两本书。感谢推荐！

**Reply by hutusi** [reply]
谢谢🤝

**Comment by Ideamaxwu** [comment]
我一直认为，理解一个理论观点，最好的方式是读一作的原文，其次好的方式是同时代的人的评价。

**Reply by hutusi** [reply]
嗯，不过有些原作太过晦涩，一些高质量的解读还是不错的，特别是哲学类的书。
